src/libraries/System.Runtime/System.Runtime.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
Same here, is this file meant to be pushed?

Reply:
No, I normally revert this but its done automatically by the IDE almost every time regardless.

Going to not revert in this case because its not worth reverting and continuing to fight against the IDE. Will also avoid us needing to respin CI.